package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon ./index.js --exec babel-node",
-    "test": " NODE_ENV=test nyc --reporter=text mocha --exit --require @babel/register --require @babel/polyfill ./tests/*",
+    "test": " NODE_ENV=test npm run migrate && NODE_ENV=test nyc --reporter=text mocha --exit --require @babel/register --require @babel/polyfill ./tests/*",
```

Comment:
I think It's not a good practice to run the migration every time you try to run a test in your development environment because migration is a single process task, it would be better if you avoid the repetition of it every time you try to run the test.

Reply:
I run migration every time I run the tests to keep the tests database updated with any changes. If there are no changes to migrate it will change nothing and if there are changes needed to migrate, then it will make sure that the tables are updated.